src/common/personalizedDigest.ts

```diff
@@ -82,7 +82,10 @@ const getPostsTemplateData = ({ posts }: { posts: TemplatePostData[] }) => {
       ),
       post_upvotes: post.upvotes || 0,
       post_comments: post.comments || 0,
-      post_summary: post.summary,
+      post_summary:
+        post.summary?.length > 150
```

Comment:
Only thing maybe, non blocking. Move `150` to a constant.

Reply:
Or even make it a prop from gb 👀👀👀👀

Reply:
![image](https://github.com/dailydotdev/daily-api/assets/9803078/8c68425b-d358-4249-911d-1943c51ebdb3)